doc/setup/deploy.rst

```diff
@@ -216,7 +216,7 @@ the :option:`normal HTTP port <odoo-bin --http-port>`
 
 .. warning:: The livechat worker requires the ``psycogreen`` Python module,
              which is not always included with all installation packages.
-             It can be manually installed with ``pip install psycogreen``.
+             It can be manually installed with ``pip3 install psycogreen``.
```

Comment:
is this warning still relevant? `psycogreen` is in the `requirements.txt` and windows build scripts so it should be included I guess.
Could you confirm @d-fence ?

Reply:
@mart-e, well, it's not always true. Actually there is no Debian package for psycogreen with python3.
So: 
* windows installer: yes
* Debian package: no
* rpm: to verify
* source: use 'pip3 install' or 'pip install' (depends if you use a virtualenv or not)

But I don't like those warnings either because they could make people think that it's the recommended (or the only) way to install the dependency. I prefer more generic warnings.